docs/cloud-dev-env.md

```diff
@@ -0,0 +1,31 @@
+# Cloud Development Environments
+
+Altis has experimental built-in support for [GitHub Codespaces](https://github.com/features/codespaces), a Cloud-based development environment, that makes it possible to spin up a complete development environment in seconds within your browser, with a full editor based on Visual Studio Code and a fully-featured environment for your project, without having to install anything on your local machine.
+
+This is useful for many reasons:
+    - Bootstrap a full development environment in seconds.
+    - Zero local installation required.
+    - Standardized and highly configurable environment setup.
+    - Powerful and scalable development environment.
+    - Possibility of sharing preview environments with colleagues.
+
+## How to start
+
+1. **Enable Codespaces support for your GitHub organization**
```

Comment:
The preview shows these headings running into the next line. I think you need to indent the next line (four spaces??) to make it honour the newline in a list.

Reply:
Interesting, local preview doesn't do that! Neither previews in comments here. Will try.